package/firmware/ipq-wifi/Makefile

```diff
@@ -2,13 +2,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/version.mk
 
 PKG_NAME:=ipq-wifi
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
Please keep the `PKG_RELEASE` at 1. This is increased when something changes in the package without changing the version, but you changed `PKG_SOURCE_DATE` so you should reset it to 1.

Reply:
Ah okey, sorry about that. Pushed new commit.